sources/packages/web/src/views/ForbiddenUser.vue

```diff
@@ -0,0 +1,10 @@
+<template>
+  <h1>Forbidden User 403</h1>
+  <h2>Please logout and try with different user type</h2>
```

Comment:
May be a good thing to have a link to logout button here.

Reply:
This is placeholder for now, I think we must design it properly in future.